moped-editor/src/views/projects/projectView/ProjectComponents/utils.js

```diff
@@ -36,6 +38,37 @@ export const getIntersectionLabel = (point, lines) => {
   return uniqueStreets.join(" / ");
 };
 
+const useStyles = makeStyles((theme) => ({
+  iconContainer: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    marginRight: theme.spacing(1),
+  },
```

Comment:
i tried coloring the icon with `color: theme.palette.primary.main` and i _think_ i prefer the visual separation between the icon and the text. curious what others think.

<img width="624" alt="Screen Shot 2022-10-31 at 1 43 36 PM" src="https://user-images.githubusercontent.com/14793120/199074358-0143a4f7-19a0-45bd-a5c8-e63497fd0688.png">

Reply:
Strong yes - blue icon 4 life.

Reply:
Ooooh yeah! I like the blue and will update. 🙏

Reply:
thanks @mddilley!